package.json

```diff
@@ -82,8 +85,6 @@
     "debounce": "^1.2.0",
     "fast-deep-equal": "2.0.1",
     "filefy": "0.1.10",
-    "jspdf": "1.5.3",
-    "jspdf-autotable": "3.5.3",
```

Comment:
Do we need to remove these? I just ran into an issue with yarn (berry):
> Module not found: material-table tried to access jsPDF, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Reply:
Ah my yarn issue turned out to be due to the dynamic `require()` stuff, so re-adding these dependencies wasn't sufficient anyway.

Reply:
I think you are right, while merging something went amiss. The idea is to have those depencies isntalled, but only require them when running withing a browser.

Reply:
resolved in
f6b13cb

Reply:
Nice one @floAr :)